exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,11 @@
 // Write your function here
 
-var greeting = createGreeting("Daniel");
 
-console.log(greeting);
+function createGreeting(a) {
```

Comment:
It's better to give the parameter a meaningful name here, so `name` instead of `a` for example